crypto/err/openssl.txt

```diff
@@ -1190,6 +1190,8 @@ RSA_R_PSS_SALTLEN_TOO_SMALL:164:pss saltlen too small
 RSA_R_PUB_EXPONENT_OUT_OF_RANGE:178:pub exponent out of range
 RSA_R_P_NOT_PRIME:128:p not prime
 RSA_R_Q_NOT_PRIME:129:q not prime
+RSA_R_RANDOMNESS_SOURCE_STRENGTH_INSUFFICIENT_FOR_KEYGEN:180:\
```

Comment:
Now that is a looong reason code name. What about just dropping the `_FOR_KEYGEN` part?

Reply:
Shortened this.